contrib/containers-common/containers-common.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash -x
+
```

Comment:
I'm not positive of the context, but I think this might be better:

```
#!/bin/bash

set -e
```

Generally/IMHO `-x` is more noisy than helpful.  Especially when you `echo "blah"` at the important points.  Not all distro's (and especially containers) have a `/bin` symlink, so it's more widely compatible to simply point there.  Lastly, I'm guessing `set -e` (exit on error) is useful here, should something like `branchversion` fail for some reason.